calculator.rb

```diff
@@ -0,0 +1,80 @@
+# Lily's calculator exercise
+
+# ------------------- methods -------------------
+def add_nums(one, two)
+  return one + two
+end
+
+def sub_nums(one, two)
+  return one - two
+end
+
+def mult_nums(one, two)
+  return one * two
+end
+
+def div_nums(one, two)
+  return one / two
+end
+
+
+# Request calculation from user
+
+puts "Hello! It is is time to calculate."
+print "Please enter an operation: "
+  calculation = gets.chomp.downcase
+
+until ["add", "+", "addition", "subtract", "-", "subtraction", "multiply", "+", "multiplication", "divide", "/", "division"].include?(calculation)
```

Comment:
Nice use of trapping user input until they enter some thing valid.  You repeat this process, so why not make it a method?  You could do something similar reading in the numbers.